api/src/controllers/publications.js

```diff
@@ -270,6 +192,72 @@ async function getGoogleScholarPublications(req, res) {
   res.status(200).json(response);
 }
 
+async function scrapeGoogleScholar(url) {
+  console.log(playwrightConfig.gScholarHome + url);
```

Comment:
Please use the `winston` logger for logging so we can inspect logfiles for debugging in AWS.